pkg/event/target/queuestore.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 const (
-	maxLimit = 10000 // Max store limit.
+	maxLimit = 1000000 // Max store limit.
```

Comment:
please set this to IntMax perhaps?

Reply:
Sorry man, I do not know Go. I'm PHP guy mostly. I thought changing 10000 to 1000000 would be OK

Reply:
@harshavardhana It gets changed to `min(limit, sys.GetMaxOpenFileLimit())` below anyway.